app/services/finance/ecf/statement_calculator.rb

```diff
@@ -148,14 +148,20 @@ def uplift_additions_count
       end
 
       def uplift_deductions_count
+        return 0 unless statement.contract.include_uplift_fees?
```

Comment:
The tests covering this and line 163 pass without the guard statement; is there a way we can isolate them in the spec?

Reply:
Not sure this is fixed? if you comment out lines 153/165 I would expect tests to fail but they don't